frontend-opal/components/WatchList1/index.js

```diff
@@ -0,0 +1,92 @@
+import React from 'react';
+import { View, Text, Image } from 'react-native';
+import { VictoryLine, VictoryChart, VictoryTheme } from 'victory-native';
+import Svg from 'react-native-svg';
+import styles from './styles';
+
+const WatchList1 = () => {
+  const data = {
+    '52 wk low': '231.6',
+    '52 wk high': '250.7',
+    'P/E': '20.3',
+    EPS: '1.2',
+    DivYield: '2.34%',
+  };
+
+  const info = [{
+    name: 'MSFT', num: '141.21', increase: '+0.23', whole: '1.2M',
+  },
+  {
+    name: 'AMZN', num: '2108.2', increase: '-0.06', whole: '609K',
+  },
+  {
+    name: 'AAPL', num: '250.7', increase: '+1.63', whole: '810K',
+  }];
+
+  const renderItems = (infos) => (
+    infos.map((info) => (
+      <View key={info.name} style={styles.info}>
+        <Text style={styles.text}>{info.name}</Text>
+        <Text style={styles.text}>{info.num}</Text>
+        <Text style={Number(info.increase) >= 0 ? styles.greenfont : styles.redfont}>
+          {info.increase}
+%
+        </Text>
+        <Text style={styles.text}>{info.whole}</Text>
+        <View style={styles.text}><Image style={styles.icon} source={require('../../assets/transformation-icon.jpg')} /></View>
+        <Image style={styles.icon} source={require('../../assets/icons-info.png')} />
+      </View>
```

Comment:
@martinupset this is a separate component

Reply:
ok